src/tensor/domain.ts

```diff
@@ -0,0 +1,130 @@
+import { PublicKey } from "@solana/web3.js";
+import { Attribute } from "./types";
+
+const LETTERS_ONLY_FORMATTED = "lettersonly";
+const DIGITS_ONLY_FORMATTED = "digitsonly";
+const PALINDROME_FORMATTED = "palindrome";
+const EMOJI_FORMATTED = "emoji";
+
+// Do these really need to be constants??
```

Comment:
yeahh pls lmk if its ok for me to just use the strgin straight up instead of the const. I'm still trying to figure out what's considered clean code